### ISSUE TITLE
[Bug]: Version 2.11.0 or later will have the problem of not refreshing the page after adding the magnet.

### ISSUE THREAD

**Issue by edu6666** [VueTorrent/VueTorrent#1854]
### Bug description

Versions after 2.11.0 (2.11.1 and 2.11.2) have a problem where the page doesn't refresh after adding a magnet link, and I need to manually refresh the page before the download page will update.

1、Add magnetism
![_20240813084335](https://github.com/user-attachments/assets/9147488d-6d16-4db8-9734-975c0a2913dc)
2、Refresh the page
![_20240813084534](https://github.com/user-attachments/assets/13f56f4c-3752-4e93-99a7-d94b55b4d5af)
3、Display
![_20240813084637](https://github.com/user-attachments/assets/a9281fc1-a2b2-4020-a835-d67649ba6d6a)
4、Delete
![_20240813084828](https://github.com/user-attachments/assets/7507ba84-3ce6-4a03-87be-07a3467379f6)
5、Still showing
![_20240813084927](https://github.com/user-attachments/assets/08b49eb2-37a1-4577-8eae-738d9d60926d)
6、Refresh the page
![_20240813085024](https://github.com/user-attachments/assets/14fa3826-e2f4-4e9f-8644-056621e9a5fb)
7、Disappear
![_20240813085131](https://github.com/user-attachments/assets/5e8aed6e-85b1-4de4-9472-27ade1f80f36)

### Expectations

Dear author, I hope you can improve this issue

### Relevant debug informations

no

### Steps to reproduce

1. ...
2. ...
...

### How did you install VueTorrent?

Downloaded from GitHub releases

### Environment description

```plain text
Device type and name: 
OS:  Chromium 118.0.5993.159

PWA: Yes / No
Running inside a container: No
```

### qBittorrent version

4.6.5.10

### VueTorrent version

2.11.1 and 2.11.2

### Agreements

- [X] The bug still occurs on the nightly build
- [X] The bug still occurs on the latest version of qBittorrent
- [X] A similar issue doesn't already exist
- [X] I checked the DevTools and included any errors that may show up

**Comment by Larsluph** [comment]
Please check your DevTools' console. You should find an error that blocks the refresh.

**Comment by edu6666** [comment]
Yes, there is no problem with version 2.11.0, version 2.11.1 and 2.11.2 have problems with PC browsers and mobile browsers.

**Comment by Larsluph** [comment]
I tried to reproduce the issue but can't succeed.

Do you see any errors in the DevTools' console? See the [wiki](https://github.com/VueTorrent/VueTorrent/wiki/Support) for help

**Comment by NewbieOrange** [comment]
@Larsluph Having the exact same issue (well, the entire torrent page is not updating at all, even download progress / speeds). I am using Firefox.

Got the following errors in console:
```
TypeError: e.value.union is not a function
    n /assets/index-CjN-J9MH.js:71
    D /assets/vue-Bn6wizpW.js:25
    M /assets/index-CjN-J9MH.js:71
    A /assets/index-CjN-J9MH.js:71
    o /assets/index-CjN-J9MH.js:69
    kD /assets/index-CjN-J9MH.js:69
    __tla /assets/index-CjN-J9MH.js:69
    BD /assets/index-CjN-J9MH.js:69
    _run /assets/index-CjN-J9MH.js:69
    PD /assets/index-CjN-J9MH.js:69
    k /assets/index-CjN-J9MH.js:69
    run /assets/vue-Bn6wizpW.js:9
    perform /assets/index-CjN-J9MH.js:69
    setInterval handler*k /assets/index-CjN-J9MH.js:32
    D /assets/vue-Bn6wizpW.js:25
    setup /assets/index-CjN-J9MH.js:770
    Pn /assets/vue-Bn6wizpW.js:13
    Wt /assets/vue-Bn6wizpW.js:13
    I /assets/vue-Bn6wizpW.js:13
    Pn /assets/vue-Bn6wizpW.js:13
    Tf /assets/vue-Bn6wizpW.js:13
    promise callback*Sf /assets/vue-Bn6wizpW.js:13
    hi /assets/vue-Bn6wizpW.js:13
    b /assets/vue-Bn6wizpW.js:13
    sl /assets/vue-Bn6wizpW.js:9
    rf /assets/vue-Bn6wizpW.js:9
    Hr /assets/vue-Bn6wizpW.js:9
    set value /assets/vue-Bn6wizpW.js:9
    o /assets/index-CjN-J9MH.js:32
    n /assets/index-CjN-J9MH.js:32
    D /assets/vue-Bn6wizpW.js:25
    M /assets/index-CjN-J9MH.js:770
    setup /assets/index-CjN-J9MH.js:770
    Rn /assets/vue-Bn6wizpW.js:13
    Pn /assets/vue-Bn6wizpW.js:13
    Wt /assets/vue-Bn6wizpW.js:13
    __weh /assets/vue-Bn6wizpW.js:13
    jr /assets/vue-Bn6wizpW.js:5
    O /assets/vue-Bn6wizpW.js:13
    run /assets/vue-Bn6wizpW.js:9
    update /assets/vue-Bn6wizpW.js:13
    $ /assets/vue-Bn6wizpW.js:13
    G /assets/vue-Bn6wizpW.js:13
    U /assets/vue-Bn6wizpW.js:13
    v /assets/vue-Bn6wizpW.js:13
    Z /assets/vue-Bn6wizpW.js:13
    mount /assets/vue-Bn6wizpW.js:13
    mount /assets/vue-Bn6wizpW.js:17
    __tla /assets/index-CjN-J9MH.js:770
    <anonymous> /assets/index-CjN-J9MH.js:770
index-CjN-J9MH.js:71:7591
    M /assets/index-CjN-J9MH.js:71
    A /assets/index-CjN-J9MH.js:71
    o /assets/index-CjN-J9MH.js:69
    kD /assets/index-CjN-J9MH.js:69
    __tla /assets/index-CjN-J9MH.js:69
    BD /assets/index-CjN-J9MH.js:69
    _run /assets/index-CjN-J9MH.js:69
    PD /assets/index-CjN-J9MH.js:69
    k /assets/index-CjN-J9MH.js:69
    run /assets/vue-Bn6wizpW.js:9
    perform /assets/index-CjN-J9MH.js:69
    (async：setInterval handler)
    k /assets/index-CjN-J9MH.js:32
    D /assets/vue-Bn6wizpW.js:25
    setup /assets/index-CjN-J9MH.js:770
    Pn /assets/vue-Bn6wizpW.js:13
    Wt /assets/vue-Bn6wizpW.js:13
    I /assets/vue-Bn6wizpW.js:13
    Pn /assets/vue-Bn6wizpW.js:13
    Tf /assets/vue-Bn6wizpW.js:13
    (async：promise callback)
    Sf /assets/vue-Bn6wizpW.js:13
    hi /assets/vue-Bn6wizpW.js:13
    b /assets/vue-Bn6wizpW.js:13
    sl /assets/vue-Bn6wizpW.js:9
    rf /assets/vue-Bn6wizpW.js:9
    Hr /assets/vue-Bn6wizpW.js:9
    set value /assets/vue-Bn6wizpW.js:9
    o /assets/index-CjN-J9MH.js:32
    n /assets/index-CjN-J9MH.js:32
    D /assets/vue-Bn6wizpW.js:25
    M /assets/index-CjN-J9MH.js:770
    setup /assets/index-CjN-J9MH.js:770
    Rn /assets/vue-Bn6wizpW.js:13
    Pn /assets/vue-Bn6wizpW.js:13
    Wt /assets/vue-Bn6wizpW.js:13
    __weh /assets/vue-Bn6wizpW.js:13
    jr /assets/vue-Bn6wizpW.js:5
    O /assets/vue-Bn6wizpW.js:13
    run /assets/vue-Bn6wizpW.js:9
    update /assets/vue-Bn6wizpW.js:13
    $ /assets/vue-Bn6wizpW.js:13
    G /assets/vue-Bn6wizpW.js:13
    U /assets/vue-Bn6wizpW.js:13
    v /assets/vue-Bn6wizpW.js:13
    Z /assets/vue-Bn6wizpW.js:13
    mount /assets/vue-Bn6wizpW.js:13
    mount /assets/vue-Bn6wizpW.js:17
    __tla /assets/index-CjN-J9MH.js:770
    <anonymous> /assets/index-CjN-J9MH.js:770
```

**Comment by Larsluph** [comment]
@NewbieOrange Thanks for the feedback, this issue has already been solved in #1838. Fix will be available in the next release. Please update to the nightly branch in the meantime.
